iib/workers/tasks/fbc_utils.py

```diff
@@ -17,6 +19,45 @@ def is_image_fbc(image):
     :return: True if image is FBC type, False otherwise (SQLite)
     :rtype: bool
     """
+    from iib.workers.tasks.utils import skopeo_inspect
+
     skopeo_output = skopeo_inspect(f'docker://{image}')
-    dc_image_label = 'operators.operatorframework.io.index.configs.v1'
-    return dc_image_label in skopeo_output['Labels']
+    fbc_image_label = 'operators.operatorframework.io.index.configs.v1'
+    return fbc_image_label in skopeo_output['Labels']
+
+
+def get_index_fbc_dir(from_index, base_dir):
+    """
+    Get file-based catalog directory from the specified index image and save it locally.
+
+    :param str from_index: index image to get file-based catalog directory from.
+    :param str base_dir: base directory to which the database file should be saved.
+    :return: path to the copied file-based catalog directory.
+    :rtype: str
+    :raises IIBError: if any podman command fails.
+    """
+    from iib.workers.tasks.build import _copy_files_from_image
```

Comment:
Here we have again private function imported to different file

Reply:
I agree with you! 
However I am not sure if I am able to refactor everything in this PR.
`_copy_files_from_image` is already imported like this in `build_regenerate_bundle.py `

Reply:
Let's address this issue in CLOUDDST-10478, OK?

Reply:
@release-engineering/exd-guild-hello-operator PTAL

Reply:
Another quick solution is to move these two functions directly to `build.py`. 
Personally I am against it because then we would mix FBC related functions with rest of  a code. Another thing is that `build.py` is already pretty big.

Reply:
FWIW, +1 to keeping fbc related functions in a different file and not bloating up build.py

Reply:
Maybe we should rather create new files for add/rm functions.. But I am fine

Reply:
IIRC: Those FBC functions should be temporary - this is solution for Phase 1. 
We might remove them in a future, I would not mix that with our code too much.